src/App.vue

```diff
@@ -1,12 +1,21 @@
 <template>
   <div id="app">
     <router-view/>
+    <page-home/>
+    <app-modal/>
   </div>
 </template>
 
 <script>
+import AppModal from './components/AppModal'
```

Comment:
please, remove it.

Reply:
In the Base app, we are using the similar approach: https://github.com/aeternity/aepp-identity/blob/c3a25cb2cb322486d1b37b6f03508e8fb50ff069/src/App.vue#L28.
Or `AppModal` is more a separate page than an overlay?